### DISCUSSION TITLE
Disconnected when I lost internet connection. Why?

### DISCUSSION THREAD

**Discussion by D4n1984** [alvr-org/ALVR#439] [General]
Hi!

I have an issue or unexpected behavior :( , but I not sure if is related to my Quest2, or ALVR, or ....

First of all, I have a single router with ONT and I can't change it because it's mandatory to connect with my phone provider.

Sometimes, I loss my internet connection a few seconds, but the wifi signal still on and Quest 2 still connected (PC use a wired connection). When this occurs, ALVR client disconnect from server ... and I can't reconnect until I close SteamVR , VR game too :( , and restart the ALVR server.

So ... has sense? ALVR dont use internet, and should being reconnect automatically.

**Comment by zmerp** [comment]
There could be a bug when the disconnection last only a small time. The next release might resolve this problem.
